.github/workflows/release.yml

```diff
@@ -5,12 +5,17 @@ on:
     tags:
       - "v*.*.*"
 
+permissions:
```

Comment:
NIT: if it is only used when creating the draft release,  then it can be configured in that particular job rather than globally

Reply:
https://github.com/elastic/apm-agent-python/pull/1814/commits/3510ffd61185615e72e8726397e26a5fed77a1a7